test/syscalls/linux/proc.cc

```diff
@@ -1908,6 +1908,23 @@ void CheckDuplicatesRecursively(std::string path) {
 
 TEST(Proc, NoDuplicates) { CheckDuplicatesRecursively("/proc"); }
 
+TEST(Proc, ReadDir) {
+  int child_pid = fork();
+  ASSERT_GE(child_pid, 0);
+  if (child_pid == 0) {
+    while(1){};
```

Comment:
We don't want to heat the Earth, pls avoid busy-loops. while (1) { sleep(1000); }